### PR TITLE
Update dark-sites.config

### PR DESCRIPTION
**Pull request by Alecks20**
Add https://alecks.dev to global dark list

### CODE REVIEW

File: src/config/dark-sites.config [hunk 1]
```diff
@@ -42,6 +42,7 @@ akkoma.dev
 albony.xyz
 alexkaehler.com
 alfawal.dev
+alecks.dev
```
Comment (Myshor) [inline src/config/dark-sites.config:45]: sites should be sorted alphabeticaly...

this site should be before `alexkaehler.com`

**Review by Myshor — CHANGES_REQUESTED** [review]

File: src/config/dark-sites.config (hunk 1 above)
Comment (Alecks20) [reply src/config/dark-sites.config]: Fixed it

**Review by Myshor — APPROVED** [review]

**Comment by Myshor** [comment]
Reverting as site has switch to dark/light mode now.

**Comment by Myshor** [comment]
Removed in #15822

**Comment by Myshor** [comment]
FYI Added detector-hint for your site in #15823

**Pull request merged**